apis/logging/v1/cluster_log_forwarder.go

```diff
@@ -33,13 +33,17 @@ func IsOutputTypeName(s string) bool {
 }
 
 // Get all subordinate condition messages for condition of type "Ready" and False
+// A 'true' Ready condition with a message means some error with pipeline but it is still valid
```

Comment:
Please investigate the kube api documentation for true/false as my recollection is that "true" means the condition applies and the message is additional information.  I can not speak to how on might further evaluate this statement in isolation without applying it to an example, but in general, if the condition is to indicate the pipeline is "degraded" then it should be 'true' plus some additional message to indicate which one.

Reply:
N/A "degraded" pipelines are invalid